### DISCUSSION TITLE
Where can I keep the auth.json file when building a webGL?

### DISCUSSION THREAD

**Discussion by sahilpesugit** [srcnalt/OpenAI-Unity#66] [Q&A]
I am using this package in a webGL build. When running in Game window it seems to work. But when building it in a webgl build I run into the error that auth.json cannot be found. What can be done to overcome this?
![image](https://github.com/srcnalt/OpenAI-Unity/assets/54506832/ea5553fc-13cb-4549-9d9f-f27028b6a866)

**Comment by sahilpesugit** [comment]
When I try to include the API key in the initialization statement, it works on the Game window again. But when built into a webGL, it throws the following error.
![image](https://github.com/srcnalt/OpenAI-Unity/assets/54506832/def1ceaf-30b0-4118-ab4e-67f899daafd1)

![image](https://github.com/srcnalt/OpenAI-Unity/assets/54506832/2d1b1f86-ef85-4414-8c9f-df1281dd5e90)

**Comment by srcnalt** [comment]
Hi @sahilpesugit,

Could you try with the latest release v0.1.14
